js/interactive-guides/step-content.js

```diff
@@ -262,6 +262,35 @@ var stepContent = (function() {
     createEndOfGuideContent();
   };
 
+  var calculateWidgetHeight = function(percentageHeight) { 
+    var widgetHeight = "300px";
+    var rightColumn = $("#code_column:visible");
+    if (rightColumn) {
+        // Get height of visible right column
+        var columnHeight = $(rightColumn).height();
```

Comment:
rightColumn is already a jQuery object, so I think you can just do 
rightColumn.height(), since there will only ever be one element under rightColumn.  Otherwise it would be
rightColumn[0].height().